src/Builder/PartialParser/ExpressionPartialParser.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\PartialParser;
+
+use ArrayObject;
+use PhpParser\Error;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\Node\Stmt\Expression;
+use PhpParser\NodeTraverser;
+use PhpParser\Parser;
+use SprykerSdk\Integrator\Builder\Visitor\ReplaceAndCollectFullyQualifiedClassNamesVisitor;
+use SprykerSdk\Integrator\Transfer\ExpressionPartialParserResultTransfer;
+
+class ExpressionPartialParser implements ExpressionPartialParserInterface
+{
+    /**
+     * @var string
+     */
+    protected const PHP_OPEN_TAG = '<?php';
+
+    /**
+     * @var \PhpParser\Parser
+     */
+    protected Parser $parser;
+
+    /**
+     * @param \PhpParser\Parser $parser
+     */
+    public function __construct(Parser $parser)
+    {
+        $this->parser = $parser;
+    }
+
+    /**
+     * @param string $codeString
+     *
+     * @return \SprykerSdk\Integrator\Transfer\ExpressionPartialParserResultTransfer
+     */
+    public function parse(string $codeString): ExpressionPartialParserResultTransfer
+    {
+        try {
+            $ast = $this->parser->parse($this->normalizeCodeString($codeString));
```

Comment:
Why `ast`?
Maybe `codeTraverse`

Reply:
It's abbreviation of `abstract syntax tree`. It's typical code parsers result.
You can find it in https://github.com/nikic/PHP-Parser#quick-start